### DISCUSSION TITLE
Data in the result grid is truncated at 100 chars

### DISCUSSION THREAD

**Discussion by boumboum** [pawelsalawa/sqlitestudio#3888] [Q&A]
### Details
Data in the result grid column displays only the first 100 chars. Is there any possibility the see the entire content of the field?
I didn't see any configuration parameter handling that point

### Operating system
Win10

### SQLiteStudio version
3.2.1

**Comment by pawelsalawa** [comment]
There are few ways to load full data:
1. If you enter editing mode, SQLiteStudio loads full data to make edition correctly.
2. You can switch to FormView - it always loads full values.

To understand why there is initial limit of 100 characters - see #3269. It also aims to address this issue better.
